### DISCUSSION TITLE
NocoDB Docker on a Digital Ocean droplet [Feb 12]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1303] [Discord Threads]
> 🙋‍♂️ **7rozen_7ear_**

**Comment by dstala** [comment]
> 👨‍🦲 **dstala**  
#### Hello <@!221325849872302081>    
#### Could you share Docker configuration for NocoDB that you have used?  
#  
> 🙋‍♂️ **7rozen_7ear_**  
#### Sure!  
[message.txt](https://cdn.discordapp.com/attachments/941623601470242868/941631391551086602/message.txt)  
#  
> 🙋‍♂️ **7rozen_7ear_**  
#### I hope this helps.  
#  
> 👨‍🦲 **dstala**  
#### I quickly tried setting up NocoDB over Digitial ocean using their Managed Apps feature. Went through fine.    
#### https://nocodb-nocodb-a344e.ondigitalocean.app/   
####    
#### If you are available, we can get on a Zoom call to check further.  
#  
> 👨‍🦲 **dstala**  
#### Steps that I had used to create Managed App, in case if you wish to try it out   
####    
#### 1. Visit https://cloud.digitalocean.com/apps/   
#### 2. Click 'Create App'   
#### 3. Choose Source: Select "Docker Hub"   
#### 4. Repository: type "nocodb/nocodb"   
#### 5. Next page, keep everything default (port number et.al)   
#### 6. Optionally, pick a name for your web-service && Region near you   
#### 7. Select plan, click 'Launch Pro App'   
####    
#### It will take a minute for setting up & provide URL for the application!  
#  
> 🙋‍♂️ **7rozen_7ear_**  
#### Alright, the Managed Apps-feature seems to be quite nifty for a quick spin.  
#  
> 🙋‍♂️ **7rozen_7ear_**  
#### But if you could have a look at my current setup on a Zoom-call, that would be kindly appreciated.  
#  
> 🙋‍♂️ **7rozen_7ear_**  
#### The idea is to selfhost our NocoDB-instance.  
#  
> 👨‍🦲 **dstala**  
#### Sure. Will DM you Zoom link.  
#  
> 🙋‍♂️ **7rozen_7ear_**  
#### Thanks!  
#  
> 🙋‍♂️ **7rozen_7ear_**  
#### Wait, can you give me 10-15 minutes?  
#  
> 👨‍🦲 **dstala**  
#### Sure.  
#  
> 🙋‍♂️ **7rozen_7ear_**  
#### Allright, I'm back and ready.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/941623601470242868)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Comment by switchupcb** [comment]
I have created a guide on how to set up NocoDB (with SSL) using Digital Ocean: https://switchupcb.com/blog/how-to-set-up-nocodb-with-ssl/
